src/main/kotlin/network/warzone/mars/player/commands/MiscCommands.kt

```diff
@@ -59,4 +64,44 @@ class MiscCommands {
         if (player == sender) sender.sendMessage("${ChatColor.GRAY}Your ping is ${ChatColor.AQUA}${ping}ms")
         else sender.sendMessage("${ChatColor.AQUA}$possessive ${ChatColor.GRAY}ping is ${ChatColor.AQUA}${ping}ms")
     }
-}
\ No newline at end of file
+
+    @Command(
+        aliases = ["tp"],
+        desc = "Teleport to a player",
+        usage = "<player> [otherPlayer]",
+        perms = ["mars.tp", "mars.tp.other"])
```

Comment:
Having two permissions nodes here means the player needs both to be able to run the command at all. It should be set to just `["mars.tp"]`